project/BuildVars.scala

```diff
@@ -2,11 +2,11 @@ import sbt._
 
 object BuildVars {
   lazy val awsVersion         = "1.11.48"
-  lazy val contentAtomVersion = "2.4.14"
+  lazy val contentAtomVersion = "2.4.16"
```

Comment:
actually don't think it should as the fields you're adding are Optional - I'll merge #108 now